### ISSUE TITLE
[BUG] Fails when styles have empty strings

### ISSUE THREAD

**Issue by Yash-Singh1** [facebook/stylex#349]
**The problem**

Currently, when you pass in an empty string to style properties, you get the following error:
```
Cannot read properties of undefined (reading 'type')
```

**How to reproduce**

Add any property other than `content` with an empty string and notice that the build errors. See the steps below for a more detailed reproduction of the issue.

Steps to reproduce:
1. Go to reproduction link: https://stackblitz.com/edit/stylex-next-ae9acp?file=app%2Fpage.tsx
2. Notice the build is currently erroring
3. Delete line 78
4. Notice the build works now

**Expected behavior**

I am not that well-versed in how StyleX works internally, but I think StyleX should apply something similar to the behavior of `null`, because an empty string usually signifies that that style is WIP. The other option is to go by the behavior of `content`, by simply adding in a:

```css
background-color: '';
```

**Environment (include versions). Did this work in previous versions?**
```
$ cat node_modules/@stylexjs/stylex/package.json | grep version

  "version": "0.4.1",
```

**Cause/Fix**

I found this issue while using the `@stylexjs/shared/lib/transformValue` library in one of my projects. I am not sure how the function works internally, but my best guess is that the problem is that StyleX is trying to parse an empty string into an AST?

**Comment by necolas** [comment]
Empty strings are usually a sign that something has gone wrong. So I don't think we'll want to treat them the same way as `null`.

> I found this issue while using the `@stylexjs/shared/lib/transformValue` library in one of my projects

Please don't reach into or depend on any internals. A future release will use npm exports to prevent this way of using the package.

**Comment by Yash-Singh1** [comment]
> Empty strings are usually a sign that something has gone wrong. So I don't think we'll want to treat them the same way as `null`.

That's possible, but I think the compiler should still provide at minimum some info giving the developer context on where the error originated from.

> Please don't reach into or depend on any internals. A future release will use npm exports to prevent this way of using the package.

No worries, I am using the internal function for a VSCode extension, not for a website I am building :)

**Comment by necolas** [comment]
Yes I think we could improve the DX with a better error message if possible

**Comment by rjsdnql123** [comment]
@nmn @necolas 
Can I work on displaying error messages for this issue using `ESLint `

**Comment by rjsdnql123** [comment]
@nmn I've attempted to resolve the issue with ESLint. Could you please review my pull request?

**Comment by nmn** [comment]
PR to make the ESLint rule catch this is released in v0.5.0.
